dbt_rpc/__main__.py

```diff
@@ -596,6 +595,13 @@ def parse_args(args, cls=DBTArgumentParser):
         expanded_user = os.path.expanduser(parsed.project_dir)
         parsed.project_dir = os.path.abspath(expanded_user)
 
+    # set_args construct a flags with command run
+    # which doesn't have defer_mode, but we need a default value
+    parsed.defer_mode = 'eager'
```

Comment:
Isn't `'eager'` for `indirect_selection`? Does this just need to be `'any_string'`?

Reply:
Ha it is one of those places I didn't clean up properly. core moved this attribute to `defer`. Removing it completely in the most recent commit, and pointed things to `defer`